stories/package.json

```diff
@@ -36,7 +36,6 @@
     "intl-dateformat": "^0.1.1",
     "memoize-one": "^5.0.2",
     "prettier": "^1.19.1",
-    "prettier-webpack-plugin": "^1.2.0",
```

Comment:
This should be removed from yarn.lock as well with a `yarn install` and then committed

Reply:
Absolutely, my bad I should have removed it.